src/lib/Core.jsx

```diff
@@ -166,6 +166,15 @@ function Core({
     );
   };
 
+  const isCorrectCheck = (index, correctAnswerIndex) => {
+    if (typeof correctAnswerIndex === 'string') {
+      return index === Number(correctAnswerIndex);
+    } if (typeof correctAnswerIndex === 'object') {
```

Comment:
do you mean else if

Reply:
Initially use else if, however eslint complained about it: "Unnecessary 'else' after 'return'.eslint[no-else-return](https://eslint.org/docs/latest/rules/no-else-return)"

Reply:
I see. because it assumes there are two cases only. I'd prefer to keep two ifs then.  please add a line break.

```jsx
const isCorrectCheck = (index, correctAnswerIndex) => {
  if (typeof correctAnswerIndex === 'string') {
    return index === Number(correctAnswerIndex);
  }

  if (typeof correctAnswerIndex === 'object') {
    return correctAnswerIndex.some((element) => element === index);
  }

  return false;
};
```